packages/utils/src/consolidated/refs-from-consolidated.ts

```diff
@@ -40,9 +40,9 @@ async function main() {
   const bundleContents = getFileContents(bundleFilePath);
   const bundle = JSON.parse(bundleContents) as Bundle<Resource>;
 
-  const { references, missingReferences } = getReferencesFromResources({
-    resources: (bundle.entry ?? [])?.flatMap(e => e.resource ?? []),
-  });
+  const entries = bundle.entry ?? [];
+  const resourcesToCheckRefs = entries.flatMap(e => e.resource ?? []);
```

Comment:
@leite08 Used the wrong terms before, but still why not `bundle.entry?.map(e => e.resource) ?? []`?

Reply:
e.resource might be undefined